test/analyticless_convergence_tests.jl

```diff
@@ -14,6 +14,8 @@ test_setup = Dict(:alg => Vern9(), :reltol => 1e-25, :abstol => 1e-25)
 sim1 = analyticless_test_convergence(dts, prob, Tsit5(), test_setup)
 sim2 = analyticless_test_convergence(dts, prob, Vern9(), test_setup)
 
+@test_throws "No analytical solution set." test_convergence(dts, prob, Tsit5())
```

Comment:
```suggestion
if VERSION >= v"1.8"
  @test_throws "No analytical solution set." test_convergence(dts, prob, Tsit5())
end
```
Ci on Julia v1.6 fails since the `@test_throws` macro has only been extended to string arguments in Julia v1.8. Thus, we need to use this test only on Julia v1.8 and newer.